### ISSUE TITLE
JavaScript error with line annotation

### ISSUE THREAD

**Issue by philippemilink** [chartjs/chartjs-plugin-annotation#339]
Hello,

I'm trying to upgrade to last version of Chart.js and this annotation plugin, and I am facing an error when trying to add a line to graph. Here is a simple script to reproduce:

```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
    </head>
    <body>
    <div style="width: 75%">
        <canvas id="canvas"></canvas>
    </div>
        <script src="node_modules/chart.js/dist/chart.js"></script>
        <script src="node_modules/chartjs-plugin-annotation/dist/chartjs-plugin-annotation.js"></script>
        <script>
            window.onload = function() {
                var ctx = document.getElementById("canvas").getContext("2d");
                window.myMixedChart = new Chart(ctx, {
                    type: "line",
                    data: {
                    labels: ["January", "February", "March"],
                    datasets: [{
                        label: "Dataset 1",
                        data: [1, 4, 9]
                    }]
                    },
                    options: {
                        responsive: true,
                        plugins: {
                            annotation: {
                                annotations: [{
                                    type: "line",
                                    mode: "vertical",
                                    scaleID: "x-axis-0",
                                    value: "February"
                                }]
                            }
                        }
                    }
                });
            };
        </script>
    </body>
</html>
```

To install dependencies:
```
yarn init
yarn add chart.js@3.0.0-beta.13
yarn add chartjs-plugin-annotation@1.0.0-beta.3
```

When loading the page, the graph doesn't appear and there is a JavaScript error:
```
Uncaught TypeError: proxy can't report a non-existent property '"borderColor"' as non-configurable
    getOwnPropertyDescriptor file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:4944
    merge file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:874
    resolveAnnotationOptions file:///.../test-chartjs/node_modules/chartjs-plugin-annotation/dist/chartjs-plugin-annotation.js:877
    beforeUpdate file:///.../test-chartjs/node_modules/chartjs-plugin-annotation/dist/chartjs-plugin-annotation.js:809
    callback file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:792
    _notify file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:5796
    notify file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:5783
    notifyPlugins file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:6812
    update file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:6455
    Chart file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:6224
    onload file:///.../test-chartjs/index.html:15
    EventHandlerNonNull* file:///.../test-chartjs/index.html:13
chart.js:4944:19
    getOwnPropertyDescriptor self-hosted:2587
    getOwnPropertyDescriptor self-hosted:2769
    getOwnPropertyDescriptor file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:4944
    merge file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:874
    resolveAnnotationOptions file:///.../test-chartjs/node_modules/chartjs-plugin-annotation/dist/chartjs-plugin-annotation.js:877
    beforeUpdate file:///.../test-chartjs/node_modules/chartjs-plugin-annotation/dist/chartjs-plugin-annotation.js:809
    callback file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:792
    _notify file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:5796
    notify file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:5783
    notifyPlugins file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:6812
    update file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:6455
    Chart file:///.../test-chartjs/node_modules/chart.js/dist/chart.js:6224
    onload file:///.../test-chartjs/index.html:15
    (Asynchrone : EventHandlerNonNull)
    <anonyme> file:///.../test-chartjs/index.html:13
``` 

If I disable the annotation, the graph appears correctly.

**Comment by kurkle** [comment]
Should be fixed by just released v1.0.0-beta.4, can you confirm?

**Comment by kurkle** [comment]
Might be actually due to the `x-axis-0` scale id, which would be `x` for chart.js v3.

**Comment by philippemilink** [comment]
> Should be fixed by just released v1.0.0-beta.4, can you confirm?

Yes, it is fixed now, thank you !



> Might be actually due to the `x-axis-0` scale id, which would be `x` for chart.js v3.

Indeed, with `x-axis-0`, I got a strange diagonal line, with `x`everything is fine.


Closing the issue, since commit aae963d9e04dda9c47965f994904596607bda952 fixes it.
